mbuild/compound.py

```diff
@@ -1138,7 +1138,159 @@ def _kick(self):
             particle.pos += (np.random.rand(3,) - 0.5) / 100
         self._update_port_locations(xyz_init)
 
-    def energy_minimization(self, steps=2500, algorithm='cg',
+    def energy_minimization(self, forcefield='UFF',steps=1000, ):
+        """Perform an energy minimization on a Compound
+
+        Utilizes Open Babel (http://openbabel.org/docs/dev/) to perform an
+        energy minimization/geometry optimization on a Compound by applying
+        a generic force field or OpenMM (http://openmm.org/) to energy minimize
+        using a user-defined forcefield xml.
```

Comment:
Should probably provide a little more description of the OpenMM usage - specifically maybe giving a link to Foyer and mentioning that this will be used for atom-typing (thus the force field XML will need to contain valid SMARTS strings)